lib/matplotlib/backends/backend_gtk3.py

```diff
@@ -169,27 +179,26 @@ def key_release_event(self, widget, event):
     def motion_notify_event(self, widget, event):
         if event.is_hint:
             t, x, y, state = event.window.get_device_position(event.device)
+            # flipy so y=0 is bottom of canvas
+            x *= self.device_pixel_ratio
+            y = (self.get_allocation().height - y) * self.device_pixel_ratio
```

Comment:
I think you can just pass `event.window.get_device_position(event.device)` (which has `x` and `y` attributes) to `_mouse_event_coords`?

Reply:
I removed that entirely, as that sort of event compression is deprecated.